commit e53881505ae212fa913fc536e60869e9b8e7f73c
Author: Ake Sandgren <ake.sandgren@hpc2n.umu.se>
Date:   Thu Oct 27 09:45:11 2022 +0200

    Don't change flags from what EasyBuild has decided to use.

diff --git xmipp-3.22.07-Helios/scripts/config.py xmipp-3.22.07-Helios/scripts/config.py
index 0f58cbe3..b442a035 100644
--- xmipp-3.22.07-Helios/scripts/config.py
+++ xmipp-3.22.07-Helios/scripts/config.py
@@ -293,7 +293,7 @@ class Config:
                 self.configDict["CCFLAGS"] += " -std=c99"
         if 'g++' in self.get(Config.KEY_CXX):
             # optimize for current machine
-            self.configDict["CXXFLAGS"] += " -mtune=native -march=native -flto"
+            #self.configDict["CXXFLAGS"] += " -mtune=native -march=native -flto"
             if "-std=c99" not in self.configDict["CXXFLAGS"]:
                 self.configDict["CXXFLAGS"] += " -std=c++17"
             if isCIBuild():
@@ -301,8 +301,8 @@ class Config:
                 self.configDict["CXXFLAGS"] += " -Werror"
                 # don't optimize, as it slows down the build
                 self.configDict["CXXFLAGS"] += " -O0"
-            else:
-                self.configDict["CXXFLAGS"] += " -O3"
+            #else:
+            #    self.configDict["CXXFLAGS"] += " -O3"
             if self.is_true("DEBUG"):
                 self.configDict["CXXFLAGS"] += " -g"
         
@@ -315,8 +315,8 @@ class Config:
         if self.configDict["LIBDIRFLAGS"] == "":
             # /usr/local/lib or /path/to/virtEnv/lib
             localLib = "%s/lib" % info['data']
-            self.configDict["LIBDIRFLAGS"] = "-L%s" % localLib
-            self.environment.update(LD_LIBRARY_PATH=localLib)
+            #self.configDict["LIBDIRFLAGS"] = "-L%s" % localLib
+            #self.environment.update(LD_LIBRARY_PATH=localLib)
 
             # extra libs
             hdf5InLocalLib = findFileInDirList("libhdf5*", localLib)
